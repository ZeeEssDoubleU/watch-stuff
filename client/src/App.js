import React, { Component } from "react";
import { Route, Switch } from "react-router-dom";

import HeaderNavContainer from "./components/HeaderNavContainer/HeaderNavContainer";
import HomeContainer from "./components/HomeContainer/HomeContainer";
import WatchContainer from "./components/WatchContainer/WatchContainer";

import "./App.scss";

const API_KEY = "AIzaSyCY1Z702IHnKeFqnF0G6xRxoLnosXwXF2Y";

class App extends Component {
	render() {
		return (
			<>
				<HeaderNavContainer />
				<div className="app-layout">
					<Switch>
						<Route exact path="/" component={HomeContainer} />
						<Route path="/watch" component={WatchContainer} />
					</Switch>
				</div>
			</>
		);
	}
}

export default App;
