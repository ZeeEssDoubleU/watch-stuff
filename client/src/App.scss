@import './styles/shared.scss';
.app-layout {
   margin-top: $header-nav-height;
   margin-left: $sidebar-left-width;

   transition: $responsive-layout-trans;

   ::-webkit-scrollbar {
      width: 6px;
   }
}