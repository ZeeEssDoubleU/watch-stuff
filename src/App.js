import React, { Component } from "react";

import HeaderNav from "./containers/HeaderNav/HeaderNav.js";
import SideBar from "./containers/SideBar/SideBar.js";

const API_KEY = "AIzaSyCY1Z702IHnKeFqnF0G6xRxoLnosXwXF2Y";

class App extends Component {
	render() {
		return (
         <div>
            <HeaderNav />
            <SideBar />
         </div>
		);
	}
}

export default App;
