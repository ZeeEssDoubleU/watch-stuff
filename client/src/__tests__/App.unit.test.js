import React from "react";
import App from "../App";
import { shallow } from "enzyme";

test("renders", () => {
	const wrapper = shallow(<App />);
	expect(wrapper).toMatchSnapshot();
});
