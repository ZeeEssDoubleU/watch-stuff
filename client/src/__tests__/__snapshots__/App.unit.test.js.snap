// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<Fragment>
  <HeaderNavContainer />
  <div
    className="app-layout"
  >
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        path="/watch"
      />
    </Switch>
  </div>
</Fragment>
`;
